src/rtc_stats.ts

```diff
@@ -47,6 +47,15 @@ export function newRTCRemoteInboundStats(report: any) {
     };
 }
 
+export function newRTCRemoteOutboundStats(report: any) {
```

Comment:
Is report typed, or can we make a type for it here?

Reply:
The only alternative would be to remove these methods altogether and use `as`. I think it would result in pretty much the same.